style/modules.css

```diff
@@ -28,23 +37,75 @@ aside {
 #projects {
   float: left;
   height:70%;
-  width: 70%;
-  margin:1em;
+  width: 80%;
+  margin-top:1em;
+  margin-bottom:1em;
+  margin-left:2em;
   background-color: #99d5cf;
   text-align:center;
 }
 
-.template {
-  display:none;
-}
 
-footer {
+
+#social {
   clear: both;
   height: 10%;
-  width: 50%;
-  margin-left:25%;
-  margin-top: -10%;
+  width: 80%;
+  margin-left:1em;
+  margin-top:8em;
   padding:.5em;
   text-align: center;
   background-color: #cceae7;
 }
+
+
+
+@media screen and (min-width:900px) {
+  body {
+    width:90%;
+    margin:auto;
+    background-color: #32ab9f;
+  }
+
+    aside {
```

Comment:
Since you gave the nav bar an ID, you should use that for styling for semantics.